src/index.js

```diff
@@ -127,9 +92,9 @@ class MyTodo {
       li.setAttribute('data-index', index);
       li.innerHTML = `
     <div class="form-check">
-    <input class="form-check-input" type="checkbox" id="${index}" ${
-  noteObj.completed ? 'checked' : ''
-} data-index=${index}>
+    <input class="form-check-input" type="checkbox" id="${index}"
+     ${noteObj.completed ? 'checked' : ''} 
+     data-index=${index}>
     <label class="form-check-label" for="${index}">
```

Comment:
- [ ] Good job so far @Yordinia, but as mentioned in the project requirements the property index should be set to the value of the new array length (i.e. if a 4th task is added to the list, the index of that task should equal 5) but in your case, the 4th element to the array has an index 3 instead of 4, kindly fix that 🙃 . The index should start with 1,2,3,... instead of 0,1,2,... 😉 

<img width="474" alt="Screenshot 2023-05-05 at 18 51 47" src="https://user-images.githubusercontent.com/74501523/236507001-993feff1-a4c1-4acb-8e05-10a3ff798788.png">